model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -940,22 +945,24 @@ case class AttributeString(value: String) extends AttributeValue
 case class AttributeNumber(value: BigDecimal) extends AttributeValue
 case class AttributeBoolean(value: Boolean) extends AttributeValue
 case class AttributeValueRawJson(value: JsValue) extends AttributeValue
-case object AttributeValueEmptyList extends AttributeList[AttributeValue] { val list = Seq.empty }
-case object AttributeEntityReferenceEmptyList extends AttributeList[AttributeEntityReference] { val list = Seq.empty }
+case object AttributeValueEmptyList extends AttributeList[AttributeValue] { val list: Seq[AttributeValue] = Seq.empty }
+case object AttributeEntityReferenceEmptyList extends AttributeList[AttributeEntityReference] {
+  val list: Seq[AttributeEntityReference] = Seq.empty
+}
 case class AttributeValueList(list: Seq[AttributeValue]) extends AttributeList[AttributeValue]
-case class AttributeEntityReferenceList(val list: Seq[AttributeEntityReference])
+case class AttributeEntityReferenceList(list: Seq[AttributeEntityReference])
     extends AttributeList[AttributeEntityReference]
-case class AttributeEntityReference(val entityType: String, val entityName: String) extends AttributeListElementable
+case class AttributeEntityReference(entityType: String, entityName: String) extends AttributeListElementable
 
 object AttributeStringifier {
   def apply(attribute: Attribute): String =
     attribute match {
       case AttributeNull                     => ""
       case AttributeString(value)            => value
       case AttributeNumber(value)            => value.toString()
-      case AttributeBoolean(value)           => value.toString()
+      case AttributeBoolean(value)           => value.toString
```

Comment:
This change confuses me. Can you explain it?

Reply:
Functionally, there's no change. 
Scala _allows_ the omission of parentheses for zero argument methods.
The convention however, is to only omit parentheses for purely functional operations, with no side effects.

This was where I let my IDE auto-apply some style inspections on the file. I think the reason it didn't remove the parentheses from the others, is that this was a built-in object library object the IDE was confident it could remove them here, but was more conservative on the others since they were things like `BigDecimal`s.